src/EditorFeatures/VisualBasicTest/Diagnostics/GenerateConstructor/GenerateConstructorTests.vb

```diff
@@ -578,6 +578,20 @@ End Class
 Public Class [|;;|]Derived
     Inherits Base
 
+End Class",
+"
+Public Class Base
+    Public Sub New(a As Integer, Optional b As String = Nothing)
+
+    End Sub
+End Class
+
+Public Class ;;Derived
+    Inherits Base
+
+    Public Sub New(a As Integer, Optional b As String = Nothing)
+        MyBase.New(a, b)
+    End Sub
 End Class")
```

Comment:
This is some fallout from my change to the semantic model. This test is from a previous bug fix (the root cause being the same issue I hit in the completion provider), but with the new semantic model behaviour, this scenario now works. I've updated the test expectations to match, but a small change to the diagnostics can restore the old behaviour if that's desired.